src/parser/extract/compiled/nodes.ts

```diff
@@ -96,7 +96,7 @@ export async function extractCompiledASTNodes(params: Params): Promise<CompiledA
         id.type === 'ObjectPattern' &&
         init?.type === 'CallExpression' &&
         init.callee.type === 'Identifier' &&
-        init.callee.name === state.defineMetaImport?.local.name
+        init.callee.name === AST_NODES_NAMES.defineMeta
```

Comment:
With this change, the user loses the ability to rename the import specifier of `defineMeta` to whatever he prefers.
For example:

```ts
import { defineMeta as dm } from "@storybook/addon-svelte-csf";
```

Why was this change necessary in your case?

Reply:
For some reason, the variable declaration appears before the import statement for me.
I now realize that the user will be unable to rename the import specifier with my approach.
Unfortunately, I'm not sure how to fix this. Maybe a double-pass approach is required?

Anyway, here is the relevant source snippet:
```svelte
<script lang="ts" context="module">
	import { defineMeta } from '@storybook/addon-svelte-csf';
	import type { Meta } from '@storybook/svelte';
	import { expect, userEvent, within } from '@storybook/test';
	import type { Addon_StoryContext, StepFunction, WebRenderer } from '@storybook/types';
	import { queries } from '@testing-library/dom';

	import TargetPicker from './TargetPicker.svelte';

	const { Story } = defineMeta({
		title: 'App/Workspace/Graph View/Node/Partials/Target Picker',
		component: TargetPicker,
		tags: ['autodocs']
	} satisfies Meta<TargetPicker>);
</script>
```

Here is the compiled output that I get:
```js
import {createHotContext as __vite__createHotContext} from "/@vite/client";
import.meta.hot = __vite__createHotContext("/src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.stories.svelte");
import "/node_modules/.cache/storybook/<hash>/sb-vite/deps/svelte_internal_disclose-version.js?v=f43de63d";

$.mark_module_start();
TargetPicker_stories[$.FILENAME] = "src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.stories.svelte";

const {Story, meta} = defineMeta({
    title: 'App/Workspace/Graph View/Node/Partials/Target Picker',
    component: TargetPicker,
    tags: ['autodocs']
});

import*as $ from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/svelte_internal_client.js?v=f43de63d";
import {defineMeta} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@storybook_addon-svelte-csf.js?v=f43de63d";
import {expect, userEvent, within} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@storybook_test.js?v=f43de63d";
import {queries} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@testing-library_dom.js?v=f43de63d";
import TargetPicker from "/src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.svelte";
```

For completeness' sake, here's the compiled output I get if I `import { defineMeta as dm }`:
```js
import {createHotContext as __vite__createHotContext} from "/@vite/client";
import.meta.hot = __vite__createHotContext("/src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.stories.svelte");
import "/node_modules/.cache/storybook/<hash>/sb-vite/deps/svelte_internal_disclose-version.js?v=f43de63d";

$.mark_module_start();
TargetPicker_stories[$.FILENAME] = "src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.stories.svelte";

const {Story, meta} = dm({
    title: 'App/Workspace/Graph View/Node/Partials/Target Picker',
    component: TargetPicker,
    tags: ['autodocs']
});

import*as $ from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/svelte_internal_client.js?v=f43de63d";
import {defineMeta as dm} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@storybook_addon-svelte-csf.js?v=f43de63d";
import {expect, userEvent, within} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@storybook_test.js?v=f43de63d";
import {queries} from "/node_modules/.cache/storybook/<hash>/sb-vite/deps/@testing-library_dom.js?v=f43de63d";
import TargetPicker from "/src/lib/components/app/workspace/graph-view/node/target-picker/TargetPicker.svelte";
```

Reply:
UPDATE: I just found out about [this](https://github.com/sveltejs/svelte/issues/13135), which is the root of the problem. Updating to the newest Svelte 5 version renders this change unnecessary.